tests/PhpCodeSnifferComponentTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace OpenEuropa\CodeReview\Tests;
```

Comment:
should we take this chance to fix the namespace, move tests under tests/src folder ?

Reply:
If we move tests the we have to fix how fixture files are loaded. They use current file location to get the path.

Reply:
out of scope so its ok, but lets create a ticket for it, maybe some day we will do it

Reply:
I've changed my mind on this, we are improving tests, moved them to src, I though was a bigger change.